src/views/TeamProgress.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div>
+    <h1>Team Progress</h1>
+    <p
+        v-if="processedTeams.length === 0"
+        class="basicText">There are no teams.</p>
+    <b-row v-else id="header" class="text-left">
```

Comment:
Can you wrape this b-row and the b-row with the for in a div and then put the v-else on the div? I think that makes it clearer that none of the table would show up